hack/multi-node/bootkube-up

```diff
@@ -20,7 +20,7 @@ fi
 
 # Render assets
 if [ ! -d "cluster" ]; then
-  ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster --api-servers=https://172.17.4.101:443 ${etcd_render_flags}
+  ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster --api-servers=https://172.17.4.101:443 ${etcd_render_flags} --experimental-calico-network-policy
```

Comment:
Was thinking about maybe add an env for this, which defaults to false. And add the experimeltal flag only when the env is set to true.
Something similar to the `SELF_HOST_ETCD`.